test/generate.test.ts

```diff
@@ -19,6 +19,25 @@ const CONFIG_WITH_AUTO_ADD_COUNT = extend({}, DEFAULT_QUERY_CONFIG, {autoAddCoun
 
 describe('generate', function () {
   describe('1D', () => {
+    it('should generate answerSet with nominalColorScaleForHighCardinality and smallBandSizeForHighCardinalityOrFacet turned off', () => {
```

Comment:
This doesn't belong here.  You're trying to test stylizer, not generic 1D logic.  

Move it to below ([here](https://github.com/uwdata/compassql/pull/187/files#r73961340)).